src/js/shared/analytics.js

```diff
@@ -0,0 +1,26 @@
+import mixpanel from "mixpanel-browser";
```

Comment:
I don't think `mixpanel-browser` can be used _directly_ in `background.js`, since it seems that `mixpanel.init()` requires the `window` object which isn't defined for service workers (sources: https://community.mixpanel.com/strategy-and-planning-9/tracking-event-from-service-worker-1686 and https://github.com/mixpanel/mixpanel-js/blob/8b2e1f7b/src/mixpanel-core.js).

Some possible workarounds could be to use the Mixpanel for NodeJS (not sure if possible, https://stackoverflow.com/questions/43684452/is-it-possible-to-require-npm-modules-in-a-chrome-extension) or to make `background.js` to send a message (using `chrome.runtime.sendMessage`) when he wants to _track_ something, and then another script (that has access to the window object, like the [browser action's popup](https://developer.chrome.com/docs/extensions/reference/browserAction/)) will listen for this message and then send the event to Mixpanel.